admin/README.md

```diff
@@ -0,0 +1,4 @@
+# solidus_admin (WIP)
+
+A Rails engine that provides a backend administration interface to the Solidus ecommerce platform.
```

Comment:
Thoughts on getting rid of the word "backend" completely? slightly_smiling_face Just saying "provides an administration interface" should be enough and less confusing.

Reply:
Sure, can I blame Copilot? 😄

Reply:
:rofl: